packages/valory/skills/decision_maker_abci/fsm_specification.yaml

```diff
@@ -42,7 +42,7 @@ transition_func:
     (BlacklistingRound, NO_MAJORITY): BlacklistingRound
     (BlacklistingRound, ROUND_TIMEOUT): BlacklistingRound
     (DecisionReceiveRound, DONE): BetPlacementRound
-    (DecisionReceiveRound, MECH_RESPONSE_ERROR): DecisionReceiveRound
+    (DecisionReceiveRound, MECH_RESPONSE_ERROR): BlacklistingRound
```

Comment:
The `MECH_RESPONSE_ERROR` event is returned if the much responds with anything that cannot be parsed in the format of `p_yes`, `p_no`, `confidence`:

https://github.com/valory-xyz/trader/blob/7be4d04e6ed847249538d4b3897372bc1d771d40/packages/valory/skills/decision_maker_abci/behaviours/decision_receive.py#L196-L199